### PR TITLE
Issue 8 create models carreer level

### LINKED ISSUE #8: Models für Karriere-Level erstellen
- [x] Erzeugen des Models Models/CareerLevel.cs mit folgenden Attributen:
    - [x] + CareerLevelID: int
    - [x] + Name: string
    - [x] + ShortName: string
    - [x] + IsActive: boolean
    - [x] + PersonsCareerLevels: ICollection<PersonsCareerLevel>
    - [x] Hinzufügen von Validierungsregeln (Vergleich [hier](https://docs.microsoft.com/de-de/aspnet/core/tutorials/first-mvc-app/validation?view=aspnetcore-3.1))

----

- [x] Erzeugen des Models Models/PersonsCareerLevel.cs mit folgenden Attributen:
    - [x] + PersonsCareerLevelID: int
    - [x] + PersonID: int
    - [x] + CareerLevelID: int
    - [x] + Begin: DateTime (Date)
    - [x] + End: DateTime (Date)
    - [x] + Person: Person
    - [x] + CareerLevel: CareerLevel
    - [x] Hinzufügen von Validierungsregeln (Vergleich [hier](https://docs.microsoft.com/de-de/aspnet/core/tutorials/first-mvc-app/validation?view=aspnetcore-3.1))

----

- [x] Attribut dem Model Models/Person.cs hinzufügen:
    - [x] + PersonsCareerLevels: ICollection<PersonsCareerLevel>

----

- [x] DbSet für in Data/ApplicationDbContext.cs anlegen
    - [x] DbSet<CareerLevel> CareerLevels
    - [x] DbSet<PersonsCareerLevel> PersonsCareerLevels
    - [x] Erstelle eine Migration:
        - `dotnet ef migrations add <hier einen namen wählen>`
        - `dotnet ef database update`

### PR DESCRIPTION
**Pull request by LTobor**
closes #8

### CODE REVIEW

**Comment by LTobor** [comment]
DataType.Date is missing by the dates. Can I just add this, migrate and update the database and push it?

**Comment by Yorreck** [comment]
> DataType.Date is missing by the dates. Can I just add this, migrate and update the database and push it?

Yes, you should do this

File: Models/PersonsCareerLevel.cs [hunk 1]
```diff
@@ -1,4 +1,5 @@
 ﻿using Microsoft.AspNetCore.Mvc;
+using Microsoft.CodeAnalysis;
```
Comment (Yorreck) [inline Models/PersonsCareerLevel.cs]: Wehre did you need this?

**Review by Yorreck — CHANGES_REQUESTED** [review]

File: Models/PersonsCareerLevel.cs (hunk 1 above)
Comment (LTobor) [reply Models/PersonsCareerLevel.cs]: I have no clue, deleted the code line.

**Review by Yorreck — APPROVED** [review]

**Pull request merged**